src/structures/ThreadChannel.js

```diff
@@ -234,6 +234,23 @@ class ThreadChannel extends Channel {
     return this.parent?.permissionsFor(memberOrRole) ?? null;
   }
 
+  /**
+   * Fetches the owner of this thread
+   * @param {FetchOwnerOptions} [options] The options for fetching the member
+   * @returns {Promise<ThreadMember>}
+   */
+  async fetchOwner({ cache = true, force = false } = {}) {
+    if (!force) {
+      const existing = this.cache.get(this.ownerId);
+      if (existing) return existing;
+    }
+
+    // We cannot fetch a single thread member, as of this commit's date, Discord API throws with 405
+    const data = await this.client.api.channels(this.thread.id, 'thread-members').get();
+    const owner = data.find(member => member.user_id === this.ownerId);
+    return this._add(owner, cache);
```

Comment:
```suggestion
    const members = await this.members.fetch(cache);
    return members.get(this.ownerId);
```
Although I'm not sure if this method is not a bit misleading without being able to actually fetch a single member.

Reply:
I used a method similar to what was used with single role fetching but yeah I know it can be a bit misleading. I'll add your suggestions